osu.Game.Rulesets.Mania/Skinning/LegacyHitExplosion.cs

```diff
@@ -66,10 +67,13 @@ private void onDirectionChanged(ValueChangedEvent<ScrollingDirection> direction)
 
         public void Animate(JudgementResult result)
         {
-            (explosion as IFramedAnimation)?.GotoFrame(0);
+            if (!(result.Judgement is HoldNoteTickJudgement))
```

Comment:
I would flip this check and early-return to reduce nesting.

Reply:
Addressed in latest commit